src/Estimators/include/BipedalLocomotion/Estimators/FloatingBaseEstimatorParams.h

```diff
@@ -0,0 +1,182 @@
+/**
+ * @file FloatingBaseEstimatorParams.h
+ * @authors Prashanth Ramadoss
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_ESTIMATORS_FBE_PARAMS_H
+#define BIPEDAL_LOCOMOTION_ESTIMATORS_FBE_PARAMS_H
+
+#include <Eigen/Dense>
+
+namespace BipedalLocomotion
+{
+namespace Estimators
+{
+namespace FloatingBaseEstimators
+{
+/**
+* @brief Struct containing sensor measurement deviation parameters of floating base estimators
+*
+*/
+class SensorsStdDev
+{
+    /**
+    * @brief Additive white Gaussian noise deviation for accelerometer measurements
+    *        Expressed in local frame as m/(s^2), in continuous time
+    */
+    Eigen::Vector3d accelerometer_noise;
+
+    /**
+    * @brief Additive white Gaussian noise deviation for gyroscope measurements
+    *        Expressed in local frame as rad/s, in continuous time
+    */
+    Eigen::Vector3d gyroscope_noise;
+
+    /**
+    * @brief Random walk bias noise deviation for accelerometer measurements
+    *        Expressed in local frame as m/s^3, in continuous time
+    */
+    Eigen::Vector3d accelerometer_bias_noise;
+
+    /**
+    * @brief Random walk bias noise deviation for gyroscope measurements
+    *        Expressed in local frame as rad/s^2, in continuous time
+    */
+    Eigen::Vector3d gyroscope_bias_noise;
+
+    /**
+    * @brief White Gaussian noise deviation for linear feet velocities in rigid contact with the environment
+    *        Expressed in local frame as m/s^2, in continuous time
+    */
+    Eigen::Vector3d contact_foot_linvel_noise;
+
+    /**
+    * @brief White Gaussian noise deviation for angular feet velocities in rigid contact with the environment
+    *        Expressed in local frame as rad/s, in continuous time
+    */
+    Eigen::Vector3d contact_foot_angvel_noise;
+
+    /**
+    * @brief White Gaussian noise deviation for linear feet velocities in swing phase
+    *        Expressed in local frame as m/s^2, in continuous time
+    */
+    Eigen::Vector3d swing_foot_linvel_noise;
+
+    /**
+    * @brief White Gaussian noise deviation for angular feet velocities in swing phase
+    *        Expressed in local frame as rad/s, in continuous time
+    */
+    Eigen::Vector3d swing_foot_angvel_noise;
+
+    /**
+    * @brief White Gaussian noise deviation for relative kinematics between IMU frame to foot contact frames
+    *        Expressed in local frame, in continuous time
+    *
+    */
+    Eigen::Matrix<double, 6, 1> forward_kinematics_noise;
+};
+
+/**
+* @brief Struct containing prior state deviation parameters of floating base estimators
+*
+*/
+struct PriorsStdDev
```

Comment:
Rename `PriorsStdDev` to `StateStdDev`

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/79/commits/f46bd5b271b6e7ea77da83c41b90178157aac069